src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -1510,6 +1685,7 @@ function tryRemoveElement(elem) {
 
         list.push('SetBrightness');
         list.push('SetAspectRatio');
+        list.push('SecondarySubtitles');
```

Comment:
WebOS should be using the custom css subtitle rendering. There is a check for `webOS` in `requiresCustomSubtitlesElement` [here](https://github.com/jellyfin/jellyfin-web/blob/5823cfcd54576b304522ca607e41465c4577e8f7/src/plugins/htmlVideoPlayer/plugin.js#L1138).

I am not sure about Tizen. Would it make sense to add it to `requiresCustomSubtitlesElement`? That seems like something that may have been missed.

I did update `renderSubtitlesWithCustomElement` to also handle a secondary subtitle track.

Reply:
I just pushed changes for the other things you pointed out. I am totally open to adding more conditions for this flag.

I assumed that the custom rendered subtitles was covering everything that doesn't support the native html ones. But now that you mention Tizen, I do see that Tizen is not in the `requiresCustomSubtitlesElement` check.

I am not too familiar with intricacies Tizen/webOS rendering since I don't have a smart tv that I can test this with.

What limits/conditions would you suggest I add?

Reply:
I checked in webOS emulator (1.2, 3.0 and 4.0). It seems to support secondary subtitles natively, but it doesn't accept CSS style for pseudo elements (`::cue`, `::-webkit-media-text-track-container` and `::-webkit-media-text-track-display`). That's probably why the native element is not used.

Real Tizen 3 and Tizen 4 support secondary subtitles natively, and CSS for pseudo elements works.

So I suppose we can leave `SecondarySubtitles` feature unconditional. Anyway, we can use a custom element if it doesn't work somewhere.

Reply:
Okay, sounds good. Thank you for checking and confirming that.